Dockerfile

```diff
@@ -5,4 +5,5 @@ LABEL maintainer="The Thanos Authors"
 
 COPY /thanos_tmp_for_docker /bin/thanos
 
+USER 1001
```

Comment:
Thanks for adding this! Just confirming, we don't need to first do `useradd`?

Reply:
@matej-g I added your suggestion.